backend/.pylintrc

```diff
@@ -0,0 +1,9 @@
+[MASTER]
```

Comment:
Also, in case you need to configure pylint make the changes in `pyproject.toml` instead for a centralised config.
Don't create a separate file.